localization/yabloc/yabloc_common/include/yabloc_common/ground_server/filter/low_pass_filter.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef YABLOC_COMMON__GROUND_SERVER__FILTER__LOW_PASS_FILTER_HPP_
+#define YABLOC_COMMON__GROUND_SERVER__FILTER__LOW_PASS_FILTER_HPP_
+
+#include <Eigen/Core>
+#include <rclcpp/time.hpp>
+#include <yabloc_common/ground_plane.hpp>
+
+#include <optional>
+
+namespace yabloc::ground_server
+{
+class LowPassFilter
```

Comment:
maybe, it is better to use [this](https://github.com/autowarefoundation/autoware.universe/tree/main/common/signal_processing) :pray:

Reply:
I have replaced lowpassfilter with your specified one in [5475827](https://github.com/autowarefoundation/autoware.universe/pull/3946/commits/5475827c7523646f19b01bfbde0c8aed9c7004df)